### DISCUSSION TITLE
Issues using Paste memo field

### DISCUSSION THREAD

**Discussion by john-aws** [FuLoRi/Analyze-LastPassVaultGUI#12] [General]
Pasting a large LastPass XML export (over 500KB) via ctrl+v truncates the pasted text at about 29k characters.

Two workarounds are available:

1. use the Paste button (rather than pressing ctrl+v in the Paste edit field)
2. use the Browse feature to supply the name of the XML file

Also, minor issue: the Paste memo field does not support ctrl+A so it is not trivial to select all text and cut/delete/replace that text. Workaround is:

1. place the insertion point after the last character of the text
2. use the mouse and scrollbar to scroll to the top of the text
3. place insertion point before the 1st character of the text and use shit+click to select all the text

This was using Windows 10 and version 1.1 of the PowerShell script.

**Comment by john-aws** [comment]
Note: I'm creating these issues primarily to help other users find workarounds if they run into the same issues. It's not to chase you to fix them. If they're trivial to fix then great, but I've provided workarounds to help others.

**Comment by FuLoRi** [comment]
Thank you for sharing these workarounds, @john-aws! Much appreciated. :)
